src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultMotion.tsx

```diff
@@ -398,8 +399,117 @@ const DefaultMotion = ({
     )} | Motion | Colony - ${colony.displayName ?? colony.colonyName ?? ``}`,
   );
 
-  // @TODO will be used in the following PR
-  // const isDecision = actionType === ColonyMotions.CreateDecisionMotion;
+  const isDecision = actionType === ColonyMotions.CreateDecisionMotion;
+  const hasBanner = useMemo(
+    () =>
+      isDecision ? false : shouldDisplayMotion(currentStake, requiredStake),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [],
+  );
+
+  // Decision specific
+  const pageFeedContent = useMemo(() => {
```

Comment:
I think that this should be a separate component with logic extracted into it. It's already a huge component/file so I thing it would be great to extract some logic/html from it.

Reply:
Sure, sounds good.